pydantic_ai_slim/pydantic_ai/models/mistral.py

```diff
@@ -79,7 +79,7 @@
     from mistralai.models.usermessage import UserMessage as MistralUserMessage
     from mistralai.types.basemodel import Unset as MistralUnset
     from mistralai.utils.eventstreaming import EventStreamAsync as MistralEventStreamAsync
-except ImportError as e:
+except ImportError as e:  # pragma: lax no cover
```

Comment:
I saw this too, it happens when extra packages are installed when we run the no-deps tests.